components/sharing/SocialShareModal.vue

```diff
@@ -5,16 +5,46 @@
     hide-header-close
     hide-footer
     data-qa="share modal"
+    v-on="onItemPage ?
+      { show: requestEmbed,
```

Comment:
This results in an attempt to make a request every time the share modal is opened. I say attempt because actually the browser appears to cache it after the first attempt.

One way around this would be to check if `this.oEmbedDataHtml` is present, and if so return early in `requestEmbed`